stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt

```diff
@@ -62,6 +64,8 @@ public object TransformStyle {
     public var typingIndicatorViewStyleTransformer: StyleTransformer<TypingIndicatorViewStyle> = noopTransformer()
     public var pinnedMessageListViewStyleTransformer: StyleTransformer<PinnedMessageListViewStyle> = noopTransformer()
     public var defaultQuotedAttachmentViewStyleTransformer: StyleTransformer<DefaultQuotedAttachmentViewStyle> = noopTransformer()
```

Comment:
I decided to proceed with 2 independent styles: `MessageComposerViewStyle` and `AttachmentsPickerDialogStyle`. Motivation: attachment picker dialog looks like independent component that should be easily replaced and therefore it deserves a separate style. As for the style of command and mention suggestion dialogs, they are an integral part of `MessageComposer` as we have content slots for them, so the style for them is incapsulated in `MessageComposerViewStyle`.